scripts/upload/add_csu_gpr_AK.py

```diff
@@ -0,0 +1,66 @@
+"""
+Read in the SnowEx 2023 CSU GPR data collected at Farmers Loop and Creamers Field.
+
+1. Data is preliminary and currently only available via email from Randall B.
+2A. python run.py # To run all together all at once
+2B. python add_gpr.py # To run individually
+
+"""
+
+from pathlib import Path
+from snowexsql.db import get_db
+from snowex_db.upload import PointDataCSV
+import pandas as pd
+
+
+def main():
+    file = Path('../download/data/SnowEx223_FLCF_1GHz_GPR_CSU.csv')
+
+    # Fix quirks
+    df = pd.read_csv(file, dtype=str)
+
+    # Upload is not able to handle the Notes col. So just remove it for now
+    modified = file.parent.joinpath(file.stem + f'_mod{file.suffix}')
+    print(f"Removing Notes Column prior to upload. Writing to {modified}")
+
+    coi = [c for c in df.columns if c != 'Notes']
+
+    # No time is a problem. Use 12 AKST == 9pm UTC
+    df['Time[HHMM]'] = '09:00'
+
```

Comment:
Note current preliminary file has Nans for time which is a no go.

Reply:
Entering 9:00 local should be fine.